aboutme/style.css

```diff
@@ -0,0 +1,56 @@
+html {
+  line-height: 1.15;
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+h1{
+  padding-top: 100px;
+  font-size:4em;
+  text-align: center;
+  margin: 0.67em 0;
+  font-family: 'Rubik Mono One', sans-serif;
+  margin: 10px;
+}
+
+h4{
+  font-family: 'Ubuntu', sans-serif;
+  text-align: center;
+}
+
+body {
+  margin: 0;
+}
+
+section{
+  font-family: 'Raleway', sans-serif;
+  padding: 20px;
+  float: left;
+  width: 50%;
+  color:gray;
+  display: inline-block;
+}
+
+aside {
+  float: right;
```

Comment:
This was an adventure to figure out :) But it looks great on your page. Remember: if you ever use float, use clear:both on the element that falls right after you want it to stop so it doesn't do crazy things (thanks footer :)